src/node_crypto.h

```diff
@@ -785,6 +785,8 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* message = nullptr);
 
+static v8::MaybeLocal<v8::Value> X509ToPEM(Environment* env, X509* cert);
```

Comment:
If this is only used within `node_crypto.cc`, there's no reason to export it in the header, just keep it local to `node_crypto.cc` in an anonymous namespace.

Reply:
`GetRootCertificates` dependencies are quite a bit further down in the file. Any preference between using a forward declaration and moving the function itself? Right now `GetRootCertificates` is near the rest of the root certificate related logic.

Reply:
a forward decl is fine but @bnoordhuis and @addaleax may have other opinions. If it makes things too difficult, feel free to ignore this and keep the decl in `node_crypto.h`

Reply:
Moved `GetRootCertificates` further down, ordered to match the `SetMethod` calls in `Initialize`. Removed the header export.

Reply:
It’s just fine to add forward declarations at the beginning of a `.cc` file too.